virtcontainers/qemu.go

```diff
@@ -509,7 +509,7 @@ func (q *qemu) createSandbox(ctx context.Context, id string, networkNS NetworkNa
 	// subsequent ones with shared=off. virtio-fs always requires shared=on for
 	// memory.
 	if q.config.SharedFS == config.VirtioFS || q.config.FileBackedMemRootDir != "" {
-		if !(q.config.BootToBeTemplate || q.config.BootFromTemplate) {
+		if (!q.config.BootToBeTemplate && !q.config.BootFromTemplate) {
```

Comment:
They're same logic with different code writting. 

What is this trying to fix?

Reply:
That's  is more readable.

Reply:
I think for this relatively simple formula, there is little difference.
If the formula were more complex, then yes, maybe [De Morgan's](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) might help us simplify things.
I'm not sure it really applies here - the risk of breaking something probably outweighs the debatable improvement in readability (yes yes, I know in this case it is pretty easy to visually check the code is functionally the same....)

Reply:
thanks for raising this @freeHackOfJeff , but I agree with @WeiZhang555 and @grahamwhaley
If we accept this change, someone else could apply `De Morgan's` again arguing readability and we'll end up with the same original expression, sorry